.gitlab-ci.yml

```diff
@@ -136,17 +139,21 @@ convert_raspbian_raspberrypi4:
 
 test_acceptance_prebuilt_raspberrypi3:
   <<: *test_acceptance_prebuilt_raspberrypi
-  dependencies:
-    - build
-    - convert_raspbian_raspberrypi3
+  needs:
+    - job: build
```

Comment:
Question:
I would have guessed that these relations are transitive, so that, since the convert job already `needs` build, is this needed here then?

Reply:
I see. I think you are right, but I don't dare to try. I will merge as-is and learn more about this feature in my playground repo next value add day. Thanks!